packages/ontario-frontend-cli/resources/templates/ontario-create-app/tests/e2e/nightwatch.conf.njk

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  src_folders: ["src/tests/e2e/**/**.spec.js"],
+
+  webdriver: {
+    start_process: true,
+    server_path: require("chromedriver").path,
+    port: 9515,
+  },
+
+  test_settings: {
+    default: {
+      desiredCapabilities: {
+        browserName: "chrome",
+      },
+    },
+  },
+};
```

Comment:
I know very little about nightwatch. I'm just wondering, is there value in doing tests in Microsoft Edge as well? Might be a future state thing, Chrome and Edge are the OPS supported browsers.